public/gallery.html

```diff
@@ -8,9 +8,9 @@
 <body class="bg-blue-50">
 <script src="https://cdn.tailwindcss.com"></script>
 <link rel="stylesheet" href="CSS/gallery.css">
-
+<div id="background-img">
```

Comment:
Again, the name is tied too much to what it is doing. 
Secondly, there are high changes there will be other background images on a website. So the name and the fac that the id is supposed to be unique don't sit right

Reply:
hero-img ? 
A hero section is suppose to be unique, that would be more appropiate i believe?